src/index.js

```diff
@@ -46,7 +46,7 @@ export default function livereload(options = { watch: '' }) {
         ? JSON.stringify(options.clientUrl)
         : process.env.CODESANDBOX_SSE
         ? `'//' + (self.location.host.replace(/^([^.]+)-\\d+/,"$1").replace(/^([^.]+)/, "$1-${port}")).split(':')[0] + '/livereload.js?snipver=1&port=443'`
-        : `'//' + (self.location.host || 'localhost').split(':')[0] + ':${port}/livereload.js?snipver=1'`
+        : `(self.location.protocol.startsWith('http') ? '' : 'http:') + '//' + (self.location.host || 'localhost').split(':')[0] + ':${port}/livereload.js?snipver=1'`
```

Comment:
What about `https`? It looks like this would force it to go over `http`.

Reply:
"https" starts with "http" so will use `//location...` which will keep the same protocol